apps/fp-frontend-app/src/behandlingsupport/melding/SettPaVentReadOnlyModal.tsx

```diff
@@ -51,7 +52,7 @@ export const SettPaVentReadOnlyModal = ({ lukkCallback, ventearsaker, ventearsak
               <FormattedMessage id="SettPaVentReadOnlyModal.Frist" />
             </Label>
             <BodyShort>
-              <DateLabel dateString={finnFrist()} />
+              <DateLabel dateString={frist || lagFramtidigFrist()} />
```

Comment:
Sonar: `Prefer using nullish coalescing operator (`??`) instead of a logical or (`||`), as it is a safer operator.`

Reply:
Så den på nett, men ikke lokal sonar i IDE.